### DISCUSSION TITLE
variable: borg--gitmodule-cache seems not correctly set

### DISCUSSION THREAD

**Discussion by lld2001** [emacscollective/borg#120] [General]
after `borg-initialize`,  the `borg--gitmodule-cache` variable is still nil.

**Comment by lld2001** [comment]
Sorry, I see. This is a private variable.

**Comment by lld2001** [comment]
I created this discussion because I wanted to optimize the following snippet code, it take 41 seconds to run.
```emacs-lisp
(dolist (drone (borg-drones))
      (dolist (p (borg-load-path drone))
        (when (file-exists-p p)
          (setq auto (car (directory-files p t ".*-autoloads.el\\'")))
          (when (and auto
                     (file-exists-p auto))
            (print auto)))))
```

**Reply by tarsius** [reply]
I might add a macro but for now searching for that variable should make it obvious how to use the cache and what is being cached.
